portality/templates/publisher/uploadmetadata.html

```diff
@@ -8,6 +8,8 @@
             <p>Uploaded files must conform to one of the approved XML formats:</p>
             <ul>
                 <li><a href="/static/doaj/doajArticles.xsd">DOAJ Native XML</a></li>
+                <li><a href="/static/doaj/crossref4.4.2.xsd">Crossref 4.4.2 XML</a></li>
```

Comment:
> I'm not sure what would happen if someone uploaded one of the examples from, say, here: https://www.crossref.org/xml-samples/ which conform to 4.3.7 rather than 4.4.2.

I've used one of these and it works ;)

> they may find it works with other versions, but we can't promise

The question is how closely we want to keep an eye on the changes in Crossref XSD schema. I would say that we should say exactly which version we support ("Crossref 4.4.2") and we cannot promise the support for the other versions. Crossref supports the whole range of documents (like, for example, books) and a lot of changes may actually do not touch articles at all and be compatible but unless we want to promise that we will check every single version of XSD I wouldn't promise it.

I haven't found the information on how breaking changes to Crossref schema are announced - maybe it would be enough to keep an eye on this.